docs/setup/setup.md

```diff
@@ -134,13 +157,23 @@ We have provided a sample node.json to add a node in kubernetes. Please make sur
       }
     }
     ```
+    note: you need to remember `metadata.name` , because edgecore need it.
 + If role is not set for the node, the pods, configmaps and secrets created/updated in the cloud cannot be synced with the node they are targeted for.
-+ Deploy node
++ Deploy node (you must run on cloud side)
```

Comment:
Nice! I think `you must run on cloud side` can add bold, **you must run on cloud side**